ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -180,6 +180,11 @@ public int getColor() {
 
   private void drawRoundedBackgroundWithBorders(Canvas canvas) {
     updatePath();
+
+    if (mPathForBorderRadius == null) {
```

Comment:
This is not a correct fix. `mPathForBorderRadius` should always be set after `updatePath()` the only reason why it's not set is because `mNeedUpdatePathForBorderRadius` is `false` which should not happen if we're updating values that impact the way we draw rounded border. So perhaps that's where the bug is